### DISCUSSION TITLE
Getting Started podinfo example

### DISCUSSION THREAD

**Discussion by haraldatbmw** [fluxcd/flux2#418] [Q&A] [answered]
For me it's not clear why the getting started podinfo example creates 3 kustomizations for the staging-cluster and only one kustomization for the production-cluster. 

What's the motivation? 
The dependency-check between the kustomizations (webapp-common <- webapp-backend <- webapp-frontend) gets lost in production?

**Comment by stefanprodan** [comment]
It shows two ways of using Flux, one is by using plain yamls with dependencies between them and the other is using a single dir kustomized up front

**Reply by haraldatbmw** [reply]
Okay, that are two ways of doing it. 
But you would not combine those two ways!? 
Wouldn't you use the one or the other for all your environments (dev, test, prod)?

**Reply by stefanprodan** [reply]
> Wouldn't you use the one or the other for all your environments (dev, test, prod)?

Yes I would.

I wrote that guide to showcase both ways, maybe it's not the best, a simpler "get started" without any deps or multi-cluster would be better.

**Reply by haraldatbmw** [reply]
Okay, now I understand. 
It would be good to add a note on the getting started page to mention this and also what's the benefits of going left or right way.

**Comment by stefanprodan** [comment, marked as answer]
@SomtochiAma simplified the get started guide, please see https://toolkit.fluxcd.io/get-started/

**Comment by somtochiama** [comment]
@haraldatbmw Any feedback/suggestions on the simplified guide would be great!

**Reply by haraldatbmw** [reply]
@SomtochiAma Looks good to me :-)
